Source/SwiftLintBuiltInRules/Rules/Lint/UnhandledThrowingTaskRule.swift

```diff
@@ -0,0 +1,238 @@
+import SwiftSyntax
+
+struct UnhandledThrowingTaskRule: ConfigurationProviderRule, SwiftSyntaxRule {
+    var configuration = SeverityConfiguration(.error)
+
+    static let description = RuleDescription(
+        identifier: "unhandled_throwing_task",
+        name: "Unhandled Throwing Task",
+        description: "Errors thrown inside this task are not handled",
```

Comment:
Is there a link for more information that you could include in this description? I feel like someone new to Swift Concurrency might not understand what the problem is or how to fix it.

E.g.

```suggestion
        description: "Errors thrown inside this task are not handled (See this forum thread for more details https://forums.swift.org/...)",
```